tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -973,20 +973,25 @@ def test_compile_module(self):
 #include <cooperative_groups.h>
 
 extern "C" __global__
-void test_grid_sync(const float* x1, const float* x2, float* y) {
+void test_grid_sync(const float* x1, const float* x2, float* y, int n) {
```

Comment:
I stared at `int n` for quite a while and realized it really meant to be used as n^2...😅